### ISSUE TITLE
Detail page go back with return key

### ISSUE THREAD

**Issue by dgomezc** [Microsoft/WindowsTemplateStudio#1890]
When enter the detail of a page or feature, to return to the previous page using keyboard navigation press the tab key until you reach the "Back" hyperlink. This action should be possible by pressing the return key only.

**Comment by sibille** [comment]
Verified in dev-nightly:
Templates version: 0.16.18046.5
Wizard version: 0.16.18046.1
